src/components/ec-modal/ec-modal.vue

```diff
@@ -0,0 +1,251 @@
+<template>
+  <transition name="ec-modal--fade">
```

Comment:
We declared previous fade transitions as elements, so this one should be `ec-modal__fade`